scrabble.js

```diff
@@ -1,8 +1,204 @@
-var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
-};
+  var prompt = require('prompt');
+  var request = require('request');
 
+// SCRABBLE
+  Scrabble = function() {
+    this._points = {
+      A: 1,
```

Comment:
Memory efficient is important and its great to start thinking about it.
Thanks.

On Wed, Jun 15, 2016 at 9:38 PM, Jennie Buechner <notifications@github.com>
wrote:

> In scrabble.js
> <https://github.com/Ada-C5/js-scrabble/pull/17#discussion_r67286415>:
>
> >
> > +// SCRABBLE
> > +  Scrabble = function() {
> > +    this._points = {
> > +      A: 1,
>
> It seems reasonable to declare this as a global constant. If you have lots
> of Scrabble objects instantiated, each one will have its own this._points
> allocated in memory. A global constant would be more memory efficient since
> there would only be one no matter how many Scrabble games there are.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ada-C5/js-scrabble/pull/17/files/68f6a6ae8a0ccd97a9fe4f3a205163ad6eefb2ee#r67286415>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AOaZbRSJ66D2CAf8K_hR6z282OqM4xFzks5qMNM4gaJpZM4Iz-uw>
> .
>